core/src/main/java/org/elasticsearch/Version.java

```diff
@@ -313,6 +313,10 @@ public String toString() {
         return sb.toString();
     }
 
+    public String displayVersion() {
```

Comment:
I pushed d57595b0fc4beeae5bb443b8387d7ff38cccf4ac, I think this will remove the confusion.